criu/pie/parasite-vdso.c

```diff
@@ -54,8 +55,19 @@ static int park_at(struct vdso_maps *rt, unsigned long vdso, unsigned long vvar)
 
 	std_log_set_gettimeofday(NULL); /* stop using vdso for timings */
 
-	if (vvar)
+	if (vvar) {
+		if (vvar_vclock_size) {
```

Comment:
@avagin Would it be possible to add a comment here is reference to the kernel commit?
`e93d2521b27f ("x86/vdso: Split virtual clock pages into dedicated mapping")`